public/lessons/www.rb

```diff
@@ -15,8 +15,7 @@ class Track
     link href: "https://www.codecademy.com/tracks/web",
          name: "Codecademy HTML & CSS Track"
 
-    project from: 'fcc',
-            name: 'Applied Responsive Web Design Projects'
+    project name: 'FreeCodeCamp "Applied Responsive Web Design Projects"'
```

Comment:
please restore the "from: fcc" version of this... then it will get an icon and stuff